applications/FluidDynamicsApplication/custom_elements/fluid_element.cpp

```diff
@@ -617,6 +617,78 @@ void FluidElement<TElementData>::CalculateGeometryData(Vector &rGaussWeights,
         rGaussWeights[g] = DetJ[g] * IntegrationPoints[g].Weight();
 }
 
+template <class TElementData>
+void FluidElement<TElementData>::Calculate(
+    const Variable<double> &rVariable,
+    double &rOutput,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    Element::Calculate(rVariable, rOutput, rCurrentProcessInfo);
+}
+
+template <class TElementData>
+void FluidElement<TElementData>::Calculate(
+    const Variable<array_1d<double, 3>> &rVariable,
+    array_1d<double, 3> &rOutput,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    Element::Calculate(rVariable, rOutput, rCurrentProcessInfo);
+}
+
+template <class TElementData>
+void FluidElement<TElementData>::Calculate(
+    const Variable<Vector >& rVariable,
+    Vector& rOutput,
+    const ProcessInfo& rCurrentProcessInfo )
+{
+    noalias( rOutput ) = ZeroVector( StrainSize );
+
+    if (rVariable == FLUID_STRESS) {
+
+        // creating a new data container that goes out of scope after the function is left
+        TElementData dataLocal;
```

Comment:
```suggestion
        TElementData data_local;
```
Snake case for local variables.

Reply:
done